frontend/src/graphql-components/tenantContext/TenantContext.tsx

```diff
@@ -0,0 +1,65 @@
+import { ApolloError } from 'apollo-client';
+import {
+  createContext,
+  FC,
+  PropsWithChildren,
+  useContext,
+  useEffect,
+  useState,
+} from 'react';
+import { AuthContext } from '../../contexts/AuthContext';
+import { TenantQuery, useTenantQuery } from '../../generated-types';
+import { updatedTenant } from '../subscription';
+
+interface ITenantContext {
+  data?: TenantQuery;
+  loading?: boolean;
+  error?: ApolloError;
+}
+
+export const TenantContext = createContext<ITenantContext>({
+  data: undefined,
+  loading: undefined,
+  error: undefined,
+});
```

Comment:
```suggestion
export const TenantContext = createContext<ITenantContext>({});
```

Reply:
I just followed what have been done in the authContext

Reply:
Guess it slipped away then, it doesn't make much sense to make an object with undefined properties... Also the interface has been defined already, it's just useless to fill in this object :)